### ISSUE TITLE
Support for screenreader usability sessions from VAOS

### ISSUE THREAD

**Issue by outerpress** [department-of-veterans-affairs/va.gov-team#55380]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [x] Something else 
 
 *Give a brief description of what your team needs support with.*

VAOS will be running usability sessions with screenreader users. We'll be testing our appointment list updates with Veterans on staging.

We need support during those sessions to help troubleshoot technical issues that users run into. We'll be aiming to recruit both experienced and new screenreader users. 

We'll be running 6-12 sessions, and would like to have the screenreader support person join them all. We tentatively plan to run sessions between March 28th and April 4th, but we may push these dates forward by a week or more.

[Research plan for more details](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/research/2023-02-appt-list-usability-screenreader/research-plan.md)

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [x] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*

TBD
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- Link to product outline: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/product/vaos-product-guide-SRT-02082022.pdf
- Link to or attach user flows: https://www.figma.com/file/xRs9s6QWoBPRhpdYCGc3cV/User-Flow?node-id=155-30914&t=vg3P6f2gopNbjL03-4
- Link to prototype or mockups: https://www.figma.com/file/JpGM8LGBCqAlL8qh3DmFk8/Home-Page-Redesign?node-id=965-53970
- Link to any epics/issues that may be helpful:
- [Research plan for more details](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/research/2023-02-appt-list-usability-screenreader/research-plan.md)

## Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

We will have completed the staging review by the time this research starts.

## When does this work need to be done?
- Estimated launch date: Early-mid April
- Estimated staging review date: March 28th
- Content and IA work needed by: N/A

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [ ] No, let's work asynchronously and meet if needed
- [x] No

The design and content has already been reviewed.

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: VAOS
-	OCTO-DE product owner: Kay Lawyer 
-	Product manager: Jeff Roof
-	Designer: Ciera Maddox and Peter RUsso 
-	FE engineer: Simi Abedowale
-	Product/team Slack channel: #vaos-team

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by sara-amanda** [comment]
## Research Support
@outerpress - Just checking in to see if you happen to have specific dates and times for your research sessions? We are looking at availability for the timespan you mention in the ticket, and we want to make sure we can participate, as it nears. _(We = @SarahKay8 @coforma-terry and myself)._ 

> We'll be running 6-12 sessions, and would like to have the screenreader support person join them all. We tentatively plan to run sessions between March 28th and April 4th, but we may push these dates forward by a week or more.

### CAIA Availability
**March 28 - April 4 accessibility availability:**

#### SK
- Unavailable 4/1-4/4
- Unavailable 3/30 and 3/31, due to another research study.
- Available 3/28 and 3/29

#### SS
- Unavailable 3/31 and 4/4, due to another research study.
- Available 3/28 (morning/afternoon), 3/29 (morning is best)
- Unavailable 3/30 and 4/3

_Thank you so much in advance!_ 

cc: @coforma-terry @SarahKay8

**Comment by outerpress** [comment]
@Cieramaddox fyi

**Comment by outerpress** [comment]
@sara-amanda my apologies I typoed - we'll be running sessions through April 14th. Could you update your availability for those dates?

**Comment by sara-amanda** [comment]
@outerpress 3/28 - 4/14?

**Comment by outerpress** [comment]
Yes, that's right @sara-amanda

**Comment by sara-amanda** [comment]
Sounds good @outerpress - thank you for confirming! I am amending our original response based on the expanded research window. 

## Research Support

### CAIA Availability
****March 28 - April 14 accessibility availability:****

#### SK
- Unavailable 4/1-4/4
- Unavailable 3/30 and 3/31, due to another research study.
**- Available 3/28 and 3/29**
**_- Will check availability for 4/5, 4/6 and 4/7_**
- Tentatively unavailable 4/10-4/14 (pending confirmation)

#### SS
- Unavailable 3/31 and 4/4, due to another research study.
**- Available 3/28 (morning/afternoon)**
**- Available 3/29 _(morning before noon EST is best)_**
- Unavailable 3/30 and 4/3
**- Available 4/5-4/7 _(mornings before noon EST)_** 
**- Available 4/10 _(morning before noon EST)_**
- Unavailable 4/11-4/11, due to PTO

cc: @coforma-terry @SarahKay8

**Comment by coforma-terry** [comment]
Hey @outerpress - Just wanted to check real quick if you had any times penned for these sessions yet? Thanks :)
CC @sara-amanda @SarahKay8

**Comment by outerpress** [comment]
@coforma-terry not yet, but Perigean has started the recruiting process so we should have some sessions scheduled soon! @Cieramaddox and I will keep you updated.

**Comment by outerpress** [comment]
@coforma-terry Perigean's scheduled our first session for tomorrow at 12pm ET but it looks like y'all aren't available that day. They're working on the rest, so more updates soon :)

**Comment by coforma-terry** [comment]
@sara-amanda @SarahKay8

**Comment by coforma-terry** [comment]
@outerpress - both @sara-amanda and @SarahKay8 don't have a free slot for tomorrow, however, @joshkimux may be able to - could you send him an invite ? Thanks!

**Comment by coforma-terry** [comment]
@outerpress - @joshkimux is maxed out as well - :(

**Comment by outerpress** [comment]
Thanks for following up @coforma-terry !  We'll do our best and let you after the session if we ran into issues

**Comment by outerpress** [comment]
@sara-amanda @SarahKay8 we have sessions scheduled! I'm assuming your calendars may have changed so please let me know which of these either of you can attend (if any) along with your email address so I can add you to the session. Many thanks!

**All times are ET**

~~P1~~
~~Thursday, March 30~~
~~12:00pm – 1:30pm~~

P2
Friday, March 31
12:00pm – 1:30pm

P3
Tuesday, April 4
12:00pm – 1:30pm

P4
Tuesday, April 4
2:00pm – 3:30pm

P5
Wednesday, April 5
12:00pm – 1:30pm

P6
Wednesday, April 5 
3:30pm – 5:00pm

P7
Thursday, April 6
1:00pm – 2:30pm

P8
Thursday, April 6
3:30pm – 5:00pm

P9
Monday, April 10
5:00pm – 6:30pm

P10
Thursday, April 13
1:00pm – 2:30pm

FYI @coforma-terry @Cieramaddox

**Comment by sara-amanda** [comment]
## Research Support

### CAIA Availability: _Sara_

@outerpress - I can be available for the following sessions. The two with the asterisks* I have a few meetings that overlap the ends of each session, but it should be ok. Just let me know which ones I should add to my calendar until the invites arrive. Thanks so much!

#### P6

- Wednesday, April 5
- 3:30pm – 5:00pm*

#### P7

- Thursday, April 6
- 1:00pm – 2:30pm*

#### P8

- Thursday, April 6
- 3:30pm – 5:00pm

#### P9

- Monday, April 10
- 5:00pm – 6:30pm

@coforma-terry @RLHecht @SarahKay8

**Comment by outerpress** [comment]
Thank you @sara-amanda! I found your Oddball address in Slack and added you to those studies 👍

**Comment by sara-amanda** [comment]
Thank you @outerpress I accepted those invites. @coforma-terry I updated the capacity calendar as well.

**Comment by joshkimux** [comment]
@outerpress I can cover P3-P5 and P10 😄

**Comment by outerpress** [comment]
Thanks @joshkimux ! I'll get you added to those in a sec

**Comment by SarahKay8** [comment]
@outerpress Good morning! I'm available for these: 


P7 Thursday, April 6 1:00pm – 2:30pm
P8 Thursday, April 6 3:30pm – 5:00pm
P10
Thursday, April 13
1:00pm – 2:30pm

**Comment by SarahKay8** [comment]
Yep please do! @outerpress the more the merrier

**Comment by joshkimux** [comment]
@outerpress [I've put together a spreadsheet for accessibility notes](https://docs.google.com/spreadsheets/d/1BvFp9mNSM_YeBQd9DIyg4ksKJKqXr2ZB32lYZgpEqg4/edit?usp=sharing). If possible, I recommend including an a11y specialist in your analysis process too. That might mean setting up additional time to meet with and review this spreadsheet with a specialist ❤️

**Comment by outerpress** [comment]
This is great, thanks @joshkimux ! I'll put some time on the calendar to review this once the sessions are close to wrapping. 

Will y'all want access to the recordings as well?

**Comment by sara-amanda** [comment]
@outerpress  Yes, please!

**Comment by outerpress** [comment]
We discussed this briefly, but just to confirm can y'all (@sara-amanda @SarahKay8 and @joshkimux) use Keybase to access recordings?

**Comment by joshkimux** [comment]
@outerpress heads up that I got a conflict for the P10 session I signed up (medical appointment)! It looks like SK will be on that one, so we should be good to go.

**Comment by outerpress** [comment]
We asked Perigean to schedule 3 more participants to try to reach a total of 5 screenreader sessions.

If anyone is available for the following dates, let me know and I'll add you to the invite:

- Tuesday the 17th at 12pm ET/9am PT
- Thursday the 20th at 4:30pm ET/ 1:30pm PT
- [TBD]

Thank you!

**Comment by outerpress** [comment]
Tuesday participant was confirmed not to be a screenreader user, so canceled. So two still TBD.

**Comment by outerpress** [comment]
Alrighty, we have two scheduled, still waiting on the third. Let me know if one of you can attend and I'll add ya to the invite @sara-amanda @SarahKay8 @joshkimux 

Both are scheduled for Thursday, April 20 (tomorrow)
- 2:15pm ET/11:15am PT
- 4:30pm ET/1:30pm PT 

Thanks!

**Comment by RLHecht** [comment]
I believe this ticket is good to close but please reopen if still needed.
